pkg/domain/infra/tunnel/volumes.go

```diff
@@ -91,3 +91,9 @@ func (ic *ContainerEngine) VolumeExists(ctx context.Context, nameOrID string) (*
 		Value: exists,
 	}, nil
 }
+
+// Volumemounted check if a given volume using plugin or filesystem is mounted or not.
+// TODO: Not used and exposed to tunnel. Will be used by `export` command which is unavailable to `podman-remote`
+func (ic *ContainerEngine) VolumeMounted(ctx context.Context, nameOrID string) (*entities.BoolReport, error) {
+	return &entities.BoolReport{Value: false}, nil
```

Comment:
This should return a error, e.g. `errors.New("not implemented")`

Reply:
@Luap99 resolved in latest commit.